packages/xrpl/src/Wallet/index.ts

```diff
@@ -201,6 +190,10 @@ class Wallet {
   /**
    * Derives a wallet from a bip39 or RFC1751 mnemonic (Defaults to bip39).
    *
+   * @deprecated since version 2.6.1.
+   * Will be deleted in version 3.0.0.
+   * Currently this implementation does not provide better reliability/Security, please use other methods to generate wallet.
```

Comment:
```suggestion
   * This representation is currently deprecated in rippled. 
   * You should use another method to represent your keys such as a seed or public/private keypair.
```

Reply:
Reliability + security aren't the main reasons we are deprecating the feature so don't want to misrepresent the problem with `fromMnemonic`.